packages/circuits/test/proveGraffitiPreimage.test.ts

```diff
@@ -61,6 +64,7 @@ const genCircuitInput = (
         graffiti: reputationRecords[attesterId].graffiti,
         sign_up: reputationRecords[attesterId].signUp,
         UST_path_elements: USTPathElements,
+        graffiti_pre_image: graffitiPreImage,
```

Comment:
you should use the `graffiti_pre_image` from the input instead of global `graffitiPreImage`.
so that you can generate a wrong graffiti pre-image

Reply:
This needs to be addressed